robotoff/slack.py

```diff
@@ -167,14 +167,15 @@ def notify_automatic_processing(self, insight: ProductInsight):
             metadata_text = f"(<{product_url}|product>, <{image_url}|source image>)"
         else:
             metadata_text = f"(<{product_url}|product>)"
+        value = insight.value or insight.value_tag
 
         if insight.type in {
             InsightType.product_weight.name,
             InsightType.expiration_date.name,
         }:
-            text = f"The {insight.type} `{insight.value}` (match: `{insight.data['raw']}`) was automatically added to product {insight.barcode}"
+            text = f"The {insight.type} `{value}` (match: `{insight.data['raw']}`) was automatically added to product {insight.barcode}"
```

Comment:
Can we make this a bit more interactive with
- a deep link to edit mode ? Currently it's only a link to the product page
- a miniature of the generating crop ?

Reply:
Normally it's prepared above in metadata_text and added here below.